### PR TITLE
[enhance](Vault) Provide guidance log when user encounters no default vault situation.

### PR DESCRIPTION
**Pull request by ByteYue**
## Proposed changes

Provide more detailed guidance log for user who might encounter no default vault situation.

### CODE REVIEW

**Comment by ByteYue** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Review by gavinchou — APPROVED** [review]

**Review by gavinchou — CHANGES_REQUESTED** [review]
need change

File: fe/fe-core/src/main/java/org/apache/doris/datasource/InternalCatalog.java [hunk 1]
```diff
@@ -2491,11 +2491,15 @@ private boolean createOlapTable(Database db, CreateTableStmt stmt) throws UserEx
                 if (info != null) {
                     storageVaultName = info.first;
                     storageVaultId = info.second;
+                } else {
+                    throw new DdlException("No Default Storage Vault. You can use show storage vault stmt to"
+                            + "get full vaults, and set one as default storage vault");
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/datasource/InternalCatalog.java:2496]:
```suggestion
                    throw new DdlException("No default storage vault."
                            + " You can use stmt `SHOW STORAGE VAULT` to get all available vaults,"
                            + " and set default vault with `SET <vault_name> AS DEFAULT STORAGE VAULT`");
```

File: fe/fe-core/src/main/java/org/apache/doris/datasource/InternalCatalog.java [hunk 2]
```diff
@@ -2491,11 +2491,15 @@ private boolean createOlapTable(Database db, CreateTableStmt stmt) throws UserEx
...
                 }
             }
 
             if (storageVaultName == null || storageVaultName.isEmpty()) {
-                throw new DdlException("Invalid Storage Vault, please set one useful storage vault");
+                throw new DdlException("Invalid Storage Vault. You can use show storage vault stmt to"
+                        + "get full vaults, and pick one to set the table's property");
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/datasource/InternalCatalog.java:2502]:
```suggestion
                throw new DdlException("Invalid Storage Vault. "
                            + " You can use `SHOW STORAGE VAULT` to get all available vaults,"
                            + ", and pick one to set the table property `\"storage_vault_name\" = \"<vault_name>\"`");
```

**Comment by gavinchou** [comment]
run buildall

**Comment by ByteYue** [comment]
run buildall

**Review by gavinchou — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**